public/manifest.json

```diff
@@ -1,3 +1,4 @@
+// This one was built by React
```

Comment:
:x: please - I think that comments aren't valid JSON, so this line would break the file.